examples/fluids/navierstokes.c

```diff
@@ -542,9 +542,9 @@ PetscErrorCode SetUpDM(DM dm, problemData *problem, const char *prefix,
     PetscFE fe;
     PetscSpace fespace;
     PetscInt ncompq = 5;
-    ierr = PetscFECreateDefault(PETSC_COMM_SELF,problem->dim, ncompq,
-                                PETSC_FALSE, prefix, PETSC_DETERMINE, &fe);
-    CHKERRQ(ierr);
+    ierr = PetscFECreateLagrange(PETSC_COMM_SELF, problem->dim, ncompq,
+                                 PETSC_FALSE, degree, PETSC_DECIDE,
+                                 &fe);
```

Comment:
Sorry, I have just noticed that the `CHKERRQ(ierr);` is missing here. Perhaps @jeremylt can add that to his active branch since he's already making some changes for the updated restrictions in this file?

Reply:
Damn; sorry. I wish I could just always stick it on the end of the line. This isn't related to restrictions so I can just fix it.

Reply:
Yeah I know it would be OT for that PR, but I don't know how to make fixes to a closed PR. Is it possible without having to open a new PR?

Reply:
You can push another commit to the branch and I can merge it to master without a PR. If these come all alone, I also don't mind committing an occasional one-liner directly on master.